lib/dl_file_uploader_worker_lib/dl_file_uploader_worker_lib/tasks/excel.py

```diff
@@ -85,18 +85,10 @@ async def run(self) -> TaskResult:
             conn: aiohttp.BaseConnector
             if self._ctx.secure_reader_settings.endpoint is not None:
                 secure_reader_endpoint = self._ctx.secure_reader_settings.endpoint
-                # TODO: after migration to aiohttp 3.9.4+ replace with
-                # ssl_context: ssl.SSLContext | True = True
-                ssl_context: Optional[ssl.SSLContext] = None
-                if self._ctx.secure_reader_settings.cafile is not None:
-                    ssl_context = ssl.SSLContext(protocol=ssl.PROTOCOL_TLS)
-                    ssl_context.load_verify_locations(cafile=self._ctx.secure_reader_settings.cafile)
-                # TODO: after migration to aiohttp 3.9.4+ replace condition with
-                # aiohttp.TCPConnector(ssl=ssl_context)
-                if ssl_context is not None:
-                    conn = aiohttp.TCPConnector(ssl=ssl_context)
-                else:
-                    conn = aiohttp.TCPConnector()
+                ssl_context: ssl.SSLContext | bool = True
+                if (cafile := self._ctx.secure_reader_settings.cafile) is not None:
+                    ssl_context = ssl.create_default_context(purpose=ssl.Purpose.SERVER_AUTH, cafile=cafile)
```

Comment:
can be replaced with
```suggestion
            ssl_context = ssl.create_default_context(purpose=ssl.Purpose.SERVER_AUTH, cafile=cafile)
```